### PR TITLE
To show users about Borders object when applied over a range of cells

### PR DESCRIPTION
**Pull request by vivekkoul1**
The original issue shows the use of Borders object over a cell. It was left to users to find out on their own how Borders behave with the range of cells. This can be pretty confusing to new learners. I tried to see how it behaves with the range of cells. The output was not what i predicted. I thought Borders object will print borders only on the outside of the range of cells since its mentioned that "Use the Borders property to return the Borders collection, which contains all four borders" in the Remarks section. Those four borders are : xlEdgeBottom, xlEdgeLeft, xlEdgeRight, xlEdgeTop. But what i saw was it applied borders on the inside as well using xlInsideHorizontal and xlInsideVertical. So it returned 6 borders. 

So if we show how Borders applies to a range of cells, then new learners do not have to assume anything. It will make this article more clear and complete.

### CODE REVIEW

**Comment by lindalu-MSFT** [comment]
Hi @vivekkoul1, you're partially right here and I understand the potential confusion to new learners. However, to add a border to a range, you use the [Range.BorderAround](https://learn.microsoft.com/office/vba/api/excel.range.borderaround) method and the [Range.Borders](https://learn.microsoft.com/office/vba/api/Excel.Range.Borders) property. I think your example of applying different xlInsideHorizontal and xlInsideVertical would be better here.

File: api/Excel.Borders.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ A collection of four **[Border](Excel.Border(object).md)** objects that represen
 
 ## Remarks
 
-Use the **Borders** property to return the **Borders** collection, which contains all four borders.
+Use the **Borders** property to return the **Borders** collection, which contains all four borders when used with a single cell. But if **Borders** property is used on a range object then **Borders** property returns **Borders** collection, which contains all four borders of a cell , plus **xlInsideHorizontal** and **xlInsideVertical** i.e. it sets inside as well as outside borders of a range selection.
```
Comment (lindalu-MSFT) [inline api/Excel.Borders.md:22]:
```suggestion
Use the **Borders** property to return the **Borders** collection, which contains all four borders. You can apply different borders to each side of a cell or range. For more information how to apply borders to a range of cells, see **[Range.Borders](Excel.Range.Borders.md)** property.
```

**Comment by vivekkoul1** [comment]
Hi @lindalu-MSFT! So nice to hear back from you. This being my first PR, I am super excited. Seems we both are on same page. Really appreciate for accepting few of my changes.

My only worry is nowhere in the documentation its mentioned that Range.Borders property sets the inside of the borders as well. Everywhere its mentioned that Range.Borders property returns only 4 borders. See the snap below fromofficial documentation.

image

My only intention is to highlight that Range.Borders property when used on a range, results in setting all 6 borders namely: xlEdgeBottom, xlEdgeLeft, xlEdgeRight, xlEdgeTop, xlInsideHorizontal, and xlInsideVertical.

When i write the below code in vba sheet:
```vb
        Dim rngInner As Range: Set rngInner = ws.Range("B2", "D4")
        
        With rngInner.Borders(xlInsideHorizontal)
          .LineStyle = xlContinuous
          .Weight = xlThin
        End With
        
        With rngInner.Borders(xlInsideVertical)
          .LineStyle = xlContinuous
          .Weight = xlThin
        End With
```
 , it sets the border as shown in the below snap:

image

But when i write this:
```vb
        Dim rng As Range: Set rng = ws.Range("B6", "D8")
        
        With rng.Borders
          .LineStyle = xlContinuous
          .Weight = xlThin
        End With
```
, it generates the following output:
image

So that means it is setting all 6 borders for a range.

Share your thoughts on this.

**Comment by vivekkoul1** [comment]
Hi @lindalu-MSFT so happy to contribute to the documentation. Really thanks for understanding during the whole process and merging my pull request. Since this was my 1st PR, many things i also didn't understand that was happening during the pull request workflow.

**Comment by vivekkoul1** [comment]
Also please let me know if i have to do anything. I have deleted my branch to avoid confusion.

**Comment by lindalu-MSFT** [comment]
You did a great job, especially the samples! We hope you continue to contribute. For faster response from me, be sure to tag me lindalu-MSFT or add me as a reviewer.

**Comment by vivekkoul1** [comment]
> You did a great job, especially the samples! We hope you continue to contribute. For faster response from me, be sure to tag me lindalu-MSFT or add me as a reviewer.

Thank you so much @lindalu-MSFT. Are you on LinkedIn? I would like to tag you in the post regarding my pull request

**Pull request merged**